shard/src/main/java/com/redhat/developer/shard/OperatorServiceInMemoryImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.redhat.developer.shard;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.developer.infra.dto.ConnectorDTO;
+import com.redhat.developer.infra.dto.ConnectorStatusDTO;
+import com.redhat.developer.ingress.IngressService;
+
+import io.quarkus.scheduler.Scheduled;
+
+@ApplicationScoped
+public class OperatorServiceInMemoryImpl implements OperatorService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OperatorServiceInMemoryImpl.class);
+
+    private List<ConnectorDTO> connectors = new ArrayList<>();
+
+    @Inject
+    ManagerSyncService managerSyncService;
+
+    //TODO: remove after we move to k8s
+    @Inject
+    IngressService ingressService;
+
+    @Override
+    public ConnectorDTO createConnectorDeployment(ConnectorDTO connector) {
+        connectors.add(connector);
+        LOGGER.info("[shard] Request deployment of Connector with ID " + connector.getId() + "... MOCKED!");
```

Comment:
Can we add some more info to the logging messages to make it easier for us to debug e.g:

`Processing deployment of Connector with id 'foo' and name 'bar' for customer 'baz'`